src/components/events/partials/ModalTabsAndPages/DetailsMetadataTab.tsx

```diff
@@ -52,6 +54,13 @@ const DetailsMetadataTab = ({
 
 	const handleSubmit = (values: { [key: string]: any }, catalog: MetadataCatalog) => {
 		dispatch(updateResource({id: resourceId, values, catalog}));
+		dispatch(addNotification({
+			type: "info",
+			key: "METADATA_SAVED",
+			duration: 3,
+			parameter: undefined,
+			context: NOTIFICATION_CONTEXT,
+		}));
 	};
```

Comment:
Wait, shouldn't something like this suffice? 👀

```suggestion
	const handleSubmit = async (values: { [key: string]: any }, catalog: MetadataCatalog) => {
		try {
			await dispatch(updateResource({id: resourceId, values, catalog}));
			dispatch(addNotification({
				type: "info",
				key: "METADATA_SAVED",
				duration: 3,
				parameter: undefined,
				context: NOTIFICATION_CONTEXT,
			}));
		} catch (_) {
			dispatch(addNotification({
				type: "warning",
				key: "METADATA_NOT_SAVED",
				duration: 3,
				parameter: undefined,
				context: NOTIFICATION_CONTEXT,
			}));
		}
	};
```

Reply:
...fair enough. My intuition told me to not do this, but now that I'm thinking about it more I'm struggling to come up with any good reasons on why this would be a bad approach.